.github/workflows/image.yaml

```diff
@@ -0,0 +1,128 @@
+# Build and push images to :
+# GitHub Container Registry (ghcr.io)
+# Red Hat Container Registry (quay.io)
+name: "Build Docker Images"
+
+on:
+  create: # Branch (so that the Docker image tags exist for a new branch when pushed)
+          # even if nothing on 'paths' below is changed.
+    branches:
+      - '**'
+
+  push:
+    paths:
+      - "**"
```

Comment:
using `**` basically means that changes to any file in this repo this + lines 15-18 seem making each other redundant